app/src/main/java/com/ayush/tictactoe/MainActivity.kt

```diff
@@ -7,138 +7,131 @@ import android.widget.Button
 import android.widget.ImageButton
 import android.widget.TextView
 import androidx.appcompat.app.AppCompatDelegate
+import com.ayush.tictactoe.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
     var player=1
+    private lateinit var binding: ActivityMainBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
 
        val boardstatus = Array(3){IntArray(3) {-1} }
 
     val msgtext=findViewById<TextView>(R.id.msgtext)
```

Comment:
enable viewBinding for msgtext

Reply:
enable viewBinding for `resetbtn` too

Reply:
Oh yeah sorry, fixed!

Reply:
Good @Animeshz and from the next time don't resolve conversations by yourself, the person who opens it has to mark it resolved.